modules/rosa-hcp-sharing-vpc-cluster-creation.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * networking/rosa-hcp-shared-vpc-config.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-sharing-vpc-cluster-creation_{context}"]
+= Step Four - Cluster Creator: Creating your cluster in a shared VPC
+To create a cluster in a shared VPC, complete the following steps.
+
+[NOTE]
+====
+Installing a cluster in a shared VPC is supported only for OpenShift 4.12.34 and later, 4.13.10 and later, and all future 4.y-streams.
+====
+
+image::372_OpenShift_on_AWS_persona_worflows_0923_4.png[]
+.Prerequisites
+
+* You have the hosted zone IDs from the *VPC Owner*.
+* You have the AWS region from the *VPC Owner*.
+* You have the subnet IDs from the *VPC Owner*.
+* You have the `Route 53 role` ARN from the *VPC Owner*.
+* You have the `VPC endpoint role` ARN from the *VPC Owner*.
+
+.Procedure
+* In a terminal, enter the following command to create the shared VPC:
++
+[source,terminal]
+----
+rosa create cluster --cluster-name <cluster_name> --sts --operator-roles-prefix <prefix> --oidc-config-id <oidc_config_id> --region us-east-1 --subnet-ids <subnet_ids> --private-hosted-zone-id <private_hosted_zone_ID> --local-hosted-zone-id <local_hosted_zone_ID> --route-53-role-arn <route_53_role_arn> vpc-endpoint-role-arn <vpc_endpoint_role_arn> --base-domain <dns-domain>
```

Comment:
Flags for `create/cluster` associated with shared-vpc:

--route53-role-arn `...` --vpc-endpoint-role-arn `...` --hcp-internal-communication-hosted-zone-id `...` --ingress-private-hosted-zone-id `...` --base-domain `...`

`route53-role-arn` is the 'new' `shared-vpc-role-arn` (`shared-vpc-role-arn` is deprecated, but still available for now, interchangeable)

`ingress-private-hosted-zone-id` is the new `private-hosted-zone-id (same deal as `route53-role-arn`, the old flag is still available but gives a deprecation warning for now)

the other 2 flags are new, and `base-domain` is the same :)

Reply:
Anywhere `route53-role-arn` is used where `shared-vpc-role-arn` was used for classic, it has been deprecated and replaced with `route53-role-arn`. But, temporarily still available and interchangeable (a warning will print when using the old name to tell users)